### DISCUSSION TITLE
Reset Server Settings?

### DISCUSSION THREAD

**Discussion by rowild** [Stability-AI/StableSwarmUI#214] [Q&A] [answered]
I wanted to change the folder for the models and used an absolut path with back slashes. This is not accepted and now the interface does not start anymore. 
What do I have to delete/change/etc in the source folder to delete those paths again?

**Comment by rowild** [comment]
And why does a path like this not work?
/Volumes/EXT 8TB - Drone/__AI/SDXL

**Comment by mcmonkey4eva** [comment, marked as answer]
You can edit `StableSwarmUI/Data/Settings.fds` in a text editor to fix it (or delete the file to full reset).

You should however go to the [Issues page](https://github.com/Stability-AI/StableSwarmUI/issues) and post whatever error you had so it can be fixed.

Any valid file path should work as long as the path exists and your user account has permission to access it.
